qiskit/transpiler/__init__.py

```diff
@@ -40,714 +41,1131 @@
 Qiskit has four pre-built transpilation pipelines available here:
 :mod:`qiskit.transpiler.preset_passmanagers`.  Unless the reader is familiar with
 quantum circuit optimization methods and their usage, it is best to use one of
-these ready-made routines.
+these ready-made routines. By default the preset pass managers are composed
+of 6 stages:
+
+#. ``init`` - This stage runs any initial passes that are run before we start embedding the
+   circuit to the backend
+#. ``layout`` - This stage runs layout and maps the virtual qubits in the circuit to the
+   physical qubits on a backend. See :ref:`layout_stage` for more details.
+#. ``routing`` - This stage runs after a layout has been run and will insert any
+   necessary gates to move the qubit states around until it can be run on
+   backend's compuling map. See :ref:`routing_stage` for more details.
```

Comment:
```suggestion
   backend's connectivity. See :ref:`routing_stage` for more details.
```
Slightly more general.

Reply:
lol, well to be fair I don't think `compuling` is even a word :P